plugins/modules/subscription_manifest.py

```diff
@@ -84,7 +84,7 @@ def main():
         supports_check_mode=False,
     )
 
-    module.task_timeout = 5 * 60
+    module.task_timeout = 10 * 60
```

Comment:
Do we want to expose that as a parameter or are we rather safe now with 10 minutes?

Reply:
I prefer to not have a param for that.

10 should be rather safe (in most cases even 5 is fine)